src/main/java/backend/Logic.java

```diff
@@ -394,6 +394,78 @@ private void revertLocalMilestoneReplace(TurboIssue originalIssue) {
     }
 
     /**
+     * Handles the result of updating an issue's state on server. Current implementation includes
+     * reverting back to the original state locally if the server update failed.
```

Comment:
Get rid of unnecessary words. e.g. 
~~`Current implementation includes reverting back to the original state locally if the server update failed.`~~ 
--> `Reverts back to the original state locally if the server update failed.`